lib/vladlev.rb

```diff
@@ -1,11 +1,12 @@
 module Vladlev
-  def self.relative_exists?(filename)
-    File.exists?(File.join(File.dirname(__FILE__), filename))
+  def self.relative_existing_path(filename)
+    file_path = File.join(File.dirname(__FILE__), filename)
+    File.exists?(file_path) && file_path
```

Comment:
`File.exists?` should be enough here. Can we change the name of this to `relative_existing_path?` to indicate that this method is a predicate? Otherwise it seems like this method returns a string path.